eternalcore/src/main/java/com/eternalcode/core/command/implementations/TpaDenyCommand.java

```diff
@@ -0,0 +1,45 @@
+package com.eternalcode.core.command.implementations;
+
+import com.eternalcode.core.chat.notification.NoticeService;
+import com.eternalcode.core.teleport.TeleportRequestManager;
+import dev.rollczi.litecommands.annotations.Execute;
+import dev.rollczi.litecommands.annotations.Permission;
+import dev.rollczi.litecommands.annotations.Section;
+import org.bukkit.entity.Player;
+import panda.std.Option;
+
+import java.util.UUID;
+
+@Section(route = "tpadeny")
+@Permission("eternalcore.command.tpadeny")
```

Comment:
brakuje usage

Reply:
ale po co usage skoro to nie ma argumentów?